README.md

```diff
@@ -76,8 +76,7 @@ While creating and maintaining a custom prelude, we are pursuing the following g
    Instead, we reexport common and well-known things from `base` and some other
    libraries that are used in everyday production programming in _Haskell_.
    > **Note**: well, we did end up inventing _some_ new things.
-4. Export more useful and commonly used functions. [Hello, my name is Dmitry. I was
-   coding _Haskell_ for 3 years but still hoogling which module `liftIO` comes from.](https://twitter.com/magnars/status/834683466130345984)
```

Comment:
This link actually works for me, does the check fail because of the redirect to x.com? I don't mind removing it, but probably `s/twitter/x` should fix it.

Reply:
Right, I could've fixed this link, but I decided to remove it because the linked post isn't even related to `universum` or Haskell.